aclk/README.md

```diff
@@ -1,10 +1,89 @@
 <!--
 ---
 title: "Agent-cloud link (ACLK)"
+description: "The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. 
+The ACLK uses MQTT over secure websockets to create, persist, and encrypt the connect from end-to-end, and enable the 
+features found in Netdata Cloud."
+date: 2020-03-18
 custom_edit_url: https://github.com/netdata/netdata/edit/master/aclk/README.md
 ---
 -->
 
 # Agent-cloud link (ACLK)
 
-This is the agent cloud link (ACLK) information file
\ No newline at end of file
+The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. The ACLK uses
+[MQTT](https://en.wikipedia.org/wiki/MQTT) over secure websockets to create, persist, and encrypt the connect from
+end-to-end, and enable the features found in Netdata Cloud.
+
+The ACLK is a vital component of the entirely redesigned and re-architected Netdata Cloud.
+
+_No data is exchanged with Netdata Cloud until you claim a node._ By claiming a node, you opt-in to sending data from
```

Comment:
I like the word "exchanged". Is it worth explicitly mentioning that we do not store metric data?

Reply:
I think that's a good idea. Could you write up something along the lines of what you're thinking? It might make sense to even explain how metrics are passed via the ACLK to a user's browser, but I'm not aware enough of how it works at this moment to accurately explain it myself.

Reply:
I’ll draft something as a comment here.

Reply:
Maybe something like this?

> Data used to render charts and alarms in Netdata Cloud is accessed directly from agents through the ACLK. Though the data does flow through Netdata servers on its way from agents to the browser, it is never logged or stored.

Reply:
I like it. Added!